cloudbuild_docker.sh

```diff
@@ -12,58 +12,30 @@ set -o xtrace
 # https://docs.docker.com/engine/reference/commandline/manifest_create/
 sed -i 's/^{/{"experimental": "enabled",/g' ~/.docker/config.json
 
-docker manifest create gcr.io/$PROJECT_ID/static:nonroot \
-   gcr.io/$PROJECT_ID/static:nonroot-amd64 \
-   gcr.io/$PROJECT_ID/static:nonroot-arm64 \
-   gcr.io/$PROJECT_ID/static:nonroot-s390x \
-   gcr.io/$PROJECT_ID/static:nonroot-ppc64le
-docker manifest push gcr.io/$PROJECT_ID/static:nonroot
-
-docker manifest create gcr.io/$PROJECT_ID/static:latest \
-   gcr.io/$PROJECT_ID/static:latest-amd64 \
-   gcr.io/$PROJECT_ID/static:latest-arm64 \
-   gcr.io/$PROJECT_ID/static:latest-s390x \
-   gcr.io/$PROJECT_ID/static:latest-ppc64le
-docker manifest push gcr.io/$PROJECT_ID/static:latest
-
-docker manifest create gcr.io/$PROJECT_ID/base:nonroot \
-   gcr.io/$PROJECT_ID/base:nonroot-amd64 \
-   gcr.io/$PROJECT_ID/base:nonroot-arm64 \
-   gcr.io/$PROJECT_ID/base:nonroot-s390x \
-   gcr.io/$PROJECT_ID/base:nonroot-ppc64le
-docker manifest push gcr.io/$PROJECT_ID/base:nonroot
-
-docker manifest create gcr.io/$PROJECT_ID/base:latest \
-   gcr.io/$PROJECT_ID/base:latest-amd64 \
-   gcr.io/$PROJECT_ID/base:latest-arm64 \
-   gcr.io/$PROJECT_ID/base:latest-s390x \
-   gcr.io/$PROJECT_ID/base:latest-ppc64le
-docker manifest push gcr.io/$PROJECT_ID/base:latest
-
-docker manifest create gcr.io/$PROJECT_ID/base:debug-nonroot \
-   gcr.io/$PROJECT_ID/base:debug-nonroot-amd64 \
-   gcr.io/$PROJECT_ID/base:debug-nonroot-arm64 \
-   gcr.io/$PROJECT_ID/base:debug-nonroot-s390x \
-   gcr.io/$PROJECT_ID/base:debug-nonroot-ppc64le
-docker manifest push gcr.io/$PROJECT_ID/base:debug-nonroot
-
-docker manifest create gcr.io/$PROJECT_ID/base:debug \
-   gcr.io/$PROJECT_ID/base:debug-amd64 \
-   gcr.io/$PROJECT_ID/base:debug-arm64 \
-   gcr.io/$PROJECT_ID/base:debug-s390x \
-   gcr.io/$PROJECT_ID/base:debug-ppc64le
-docker manifest push gcr.io/$PROJECT_ID/base:debug
-
-docker manifest create gcr.io/$PROJECT_ID/cc:latest \
-   gcr.io/$PROJECT_ID/cc:latest-amd64 \
-   gcr.io/$PROJECT_ID/cc:latest-arm64
-docker manifest push gcr.io/$PROJECT_ID/cc:latest
-
-docker manifest create gcr.io/$PROJECT_ID/cc:debug \
-   gcr.io/$PROJECT_ID/cc:debug-amd64 \
-   gcr.io/$PROJECT_ID/cc:debug-arm64
-docker manifest push gcr.io/$PROJECT_ID/cc:debug
-
+docker_manifest() {
+  image=$1
+  from_images=""
+
+  for arch in $ARCHS; do
```

Comment:
Why is it that you pass "image" as a function arg while "arch" as a variable? I'd go for function args, e.g.,
```
docker_manifest gcr.io/$PROJECT_ID/static${distro_suffix}:nonroot "amd64 arm64 s390x ppc64le"
```

Reply:
Sure. I updates it.